apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceExpectationsTest.kt

```diff
@@ -37,5 +51,8 @@ class CharSequenceExpectationsTest : AbstractCharSequenceExpectationsTest(
 
         a1 = a1.toMatch(Regex(".+Robert"))
         a1 = a1.notToMatch(Regex("a"))
+
+        a1 = a1.length { toEqual(23) }
```

Comment:
something I overlooked before
```suggestion
        val l1: Expect<Int> = a1.length
        val l2: Expect<Int> = a2.length
```